### PR TITLE
Viz

### PR DESCRIPTION
**Pull request by achinth-b**
Added the viz.js file  in VizScreen
Fixed submodule problems caused by git init

### CODE REVIEW

File: frontend/App.js [hunk 1]
```diff
@@ -1,19 +1,10 @@
-import React from 'react';
```
Comment (tarikeshaq) [inline frontend/App.js:1]: Can't seem to get the viz to render on my Android or web, might need to look into that?

File: frontend/App.js [hunk 2]
```diff
@@ -1,19 +1,10 @@
...
-import { StyleSheet, Text, View } from 'react-native';
+import React from "react";
+import Viz from "./screens/DataScreen/viz";
```
Comment (tarikeshaq) [inline frontend/App.js]: Should this be "./screens/VizScreen/viz

Either way we'll move this to ProductInfo later, but for now ;)

**Review by tarikeshaq — CHANGES_REQUESTED** [review]
I can't seem to get it to render on my Android phone or on web (i.e "expo start --web"). Might need to look into this :)

File: frontend/App.js (hunk 2 above)
Comment (sarahdagger) [reply frontend/App.js]: Yes, please move your code from viz.js to HomeScreen/index.js and change that import to "./screens/HomeScreen". Then it should work and be in the Home screen, where we'll want to display it in the app :)

File: frontend/App.js (hunk 1 above)
Comment (sarahdagger) [reply frontend/App.js:1]: It works on my iPhone (with the import fix mentioned below), but not on the web for me either. I'll look into it, but if we can't fix it by Saturday, we can always just demo it on an iPhone or an emulator.

File: frontend/App.js (hunk 2 above)
Comment (achinth-b) [reply frontend/App.js]: Ok I will have it moved to the HomeScreen, and I will be looking to try to fix this web issue tonight.

**Pull request closed without merging**